app/views/editor/cinematic/CinematicEditorComponent.vue

```diff
@@ -101,13 +103,17 @@ module.exports = Vue.extend({
           }
         })
       }
-
       const c = this.cinematic
       const data = $.extend(true, {}, c.attributes)
       const el = $(`<div></div>`);
+      const files = new CocoCollection(await api.files.getDirectory({path: 'cinematic'}), { model: Cinematic })
```

Comment:
I'm always slightly thrown off when I see `await` within another construct like parenthesis. Bringing this up here as a question: Do you (or anyone else) get that feeling too? Like you have to check that there isn't a subtle issue hiding between the `await` and the `()`, for example?

Reply:
Yeah. I agree with you. This needs to be clearer.

Reply:
@differentmatt What do you think of this pattern?